submissions/BogdanGapon/HTML & CSS popup/index.html

```diff
@@ -0,0 +1,184 @@
+                      height="45"
+                    />Calendar</a
+                  >
+                  <a href="" class="popup-list__link"
+                    ><img
+                      src="./img/chat.png"
+                      alt="Chat"
+                      width="45"
+                      height="45"
+                    />Chat</a
+                  >
+                  <a href="" class="popup-list__link"
+                    ><img
+                      src="./img/classroom.png"
+                      alt="Classroom"
+                      width="45"
+                      height="45"
+                    />Classroom</a
+                  >
+                  <a href="" class="popup-list__link"
+                    ><img
+                      src="./img/contacts.png"
+                      alt="Contacts"
+                      width="45"
+                      height="45"
+                    />Contacts</a
+                  >
+                  <a href="" class="popup-list__link"
+                    ><img
+                      src="./img/documents.png"
+                      alt="Documents"
+                      width="45"
+                      height="45"
+                    />Documents</a
+                  >
+                  <a href="" class="popup-list__link"
+                    ><img
+                      src="./img/files.png"
+                      alt="Files"
+                      width="45"
+                      height="45"
+                    />Files</a
+                  >
+                  <a href="" class="popup-list__link"
+                    ><img
+                      src="./img/mail.png"
+                      alt="Mail"
+                      width="45"
+                      height="45"
+                    />Mail</a
+                  >
+                  <a href="" class="popup-list__link"
+                    ><img
+                      src="./img/maps.png"
+                      alt="Maps"
+                      width="45"
+                      height="45"
+                    />Maps</a
+                  >
+                  <a href="" class="popup-list__link"
+                    ><img
+                      src="./img/mars.png"
+                      alt="Mars"
+                      width="45"
+                      height="45"
+                    />Mars</a
+                  >
+                  <a href="" class="popup-list__link"
+                    ><img
+                      src="./img/photos.png"
+                      alt="Contacts"
+                      width="45"
+                      height="45"
+                    />Contacts</a
+                  >
+                </div>
+
+                <input
+                  class="checkbox-more visually-hidden"
+                  type="checkbox"
+                  name="checkbox_2"
+                  id="labe_id"
+                />
+                <label for="labe_id" class="label-more is-hidden"> More </label>
```

Comment:
Please, delete white spaces. `<label for="labe_id" class="label-more is-hidden">More</label>`